guides/17_Migration_to_3_0.md

```diff
@@ -162,6 +162,51 @@ handle them please remove.
 - StateMachineTransitionFailed
 - AudioDeviceSwitched
 
+## AudioVideo events
+
+The following `AudioVideo` events have been removed in v3.x, these events redirect data from `metricDidReceive`.
+
+- videoSendHealthDidChange
+- videoSendBandwidthDidChange
+- videoReceiveBandwidthDidChange
+
+In 3.x you can still replecate these events funtionality through `metricDidReceive`.
```

Comment:
Should we mention what does the code looks like before to compare them.

Reply:
I don't think its needed but I also don't think it hurts to have. Updated

Reply:
Just want to say that you don't have to make changes if you don't think it's needed. It's fair to push back. 😄

Thanks for adding it, it's not necessary but good to have.